mock-core-configs/etc/mock/templates/almalinux-8.tpl

```diff
@@ -3,7 +3,7 @@ config_opts['dist'] = 'el8.alma'  # only useful for --resultdir variable subst
 config_opts['releasever'] = '8'
 config_opts['package_manager'] = 'dnf'
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
-config_opts['bootstrap_image'] = 'docker.io/almalinux/almalinux:8'
+config_opts['bootstrap_image'] = 'docker.io/library/almalinux:8'
```

Comment:
I would suggest using `quay.io/almalinux/almalinux:8` instead. [It's an official image](https://quay.io/repository/almalinux/almalinux) and updated more frequently.

Reply:
> I would suggest using `quay.io/almalinux/almalinux:8` instead. [It's an official image](https://quay.io/repository/almalinux/almalinux) and updated more frequently.

I just did. We could also switch to `quay.io/almalinux/8-base` which is UBI-compatible. Then, we would be in line with `rhel8.tpl`. What do you think?

Reply:
That would probably work too.